docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: getting-started
+slug: /cloud/nexus/getting-started
+title: Getting Started with Temporal Nexus
+description: Get started with Temporal Nexus, including setting up Nexus Endpoints and integrating Nexus into your Temporal Workflows.
+sidebar_label: Getting Started
+tags:
+  - temporal cloud
+  - nexus setup
+keywords:
+  - Temporal Nexus onboarding
+  - setting up Nexus Endpoints
+  - temporal cloud
+  - nexus setup
+
+---
+
+Temporal Nexus Public Preview is available within your Temporal Cloud account, with support in the [Temporal Go SDK](/develop/go).
+
+Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus API Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
+
+Monolithic Namespaces can be decomposed into multiple Namespaces, by hiding service implementations behind a Nexus Endpoint in the monolithic Namespace, pointing all consumers at the new Nexus Endpoint, and then changing the Endpoint’s target Namespace to a different Namespace. Multiple Nexus Endpoints can target a single monolithic Namespace.
```

Comment:
This is really confusing and I don't understand the why behind this. Is the goal that there's one entry point that can be broken down to simple components and treated as a single endpoint, so you don't  have to use a monolithic implementation? I _think_ this is sort of what you're aiming for.

```suggestion
Monolithic Namespaces can be decomposed into multiple Namespaces by hiding service implementations behind  Nexus Endpoints.
This points consumers to a single Nexus Endpoint, redirecting work to a specialized Namespace.
This allows each subordinate Namespace to specialize its functionality without creating monoliths.
Nexus Endpoint redirects are not exclusive.
More than one Nexus Endpoint can redirect to the same subordinate Namespace, allowing Namespace functionality to be shared between Nexus solutions.
```